package.json

```diff
@@ -124,7 +124,7 @@
   },
   "prettier": "@prisma-labs/prettier-config",
   "peerDependencies": {
-    "@prisma/client": "^3.10",
+    "@prisma/client": "*",
```

Comment:
Should this be an asterisk or `^4.0` like we discussed last week?

Reply:
yes, I will update it as we discussed last week